girder_nwb/plugin_registry.py

```diff
@@ -0,0 +1,21 @@
+import importlib
+
+import pluggy
+import hookspecs
+
+
+default_plugins = ("visual_behavior", "visual_coding")
+
+
+def register_plugins(plugin_manager):
+    plugin_manager.add_hookspecs(hookspecs)
+    for i in default_plugins:
+        plugin = importlib.import_module(".{}".format(i), package="girder_nwb.plugins")
```

Comment:
I think this sets a bad example.  I would just import these modules using regular `import` statements.

Reply:
I agree. I saw this guy was doing it that way. http://devork.be/talks/pluggy/pluggy.org.
Also pytest had https://github.com/pytest-dev/pytest/blob/9cde67c09c5e980f0750edc07cb398acf9d244cf/src/_pytest/config/__init__.py#L111-L137 but we will never ever have that many plugins. So it makes perfect sense to change it.